CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ BUG FIXES:
 ENHANCEMENTS:
 * Update API doc links from terraform.io to developer.hashicorp domain by @uk1288 [#764](https://github.com/hashicorp/terraform-provider-tfe/pull/764)
 * Update website docs to depict the use of set with `tfe_team_organization_members` and `tfe_team_members` by @uk1288 [#767](https://github.com/hashicorp/terraform-provider-tfe/pull/767)
-* d/tfe_workspace: Add `execution_mode` field to workspace datasoure @Uk1288 ([#772](https://github.com/hashicorp/terraform-provider-tfe/pull/772))
+* d/tfe_workspace: Add `execution_mode` field to workspace datasoure by @uk1288 ([#772](https://github.com/hashicorp/terraform-provider-tfe/pull/772))
+* r/tfe_workspace: `vcs_repo` is now marked as computed, allowing for manual VCS configuration by @sebasslash ([#777](https://github.com/hashicorp/terraform-provider-tfe/pull/777))
```

Comment:
Mmm, this description is a bit too "inside baseball" for normal users, though it'd make sense to veteran provider developers. How bout,

```suggestion
* r/tfe_workspace: If `vcs_repo` isn't specified in the config, allow manually connecting a VCS repo without reporting it as a diff by @sebasslash ([#777](https://github.com/hashicorp/terraform-provider-tfe/pull/777))
```